sky/cli.py

```diff
@@ -1699,25 +1716,34 @@ def autostop(
     # NOTE(dev): Keep the docstring consistent between the Python API and CLI.
     """Schedule or cancel an autostop or autodown for cluster(s).
```

Comment:
The replacement seems not including the ability to cancel the autostop?

Reply:
> The replacement seems not including the ability to cancel the autostop?

That's an optional feature, the initial sentence should indicate the default behavior, ideally high level. Example: `man rsync` even if it has lots of smaller features like symbolic links. So, agreed with @concretevitamin here.

> Current wording of "scheduling" reads fine to me, and is used in many places already:

Ah i was initally confused since wasn't clear if `autostop` or `autodown` was a noun (it is typically a verb, not as a gerund). Better suggestion: `Schedules an automatic stopping or downing for cluster(s)`.

Reply:
The nouns have been advertised in many places. No users complained so far. And they sound catchy. Let's keep.

Reply:
> I'm thinking some of our help strs may be a bit too detailed. Thus, opted to not repeat the cancel sentence here as it's already documented for the --cancel arg. Wdyt?
> That's an optional feature, the initial sentence should indicate the default behavior, ideally high level. Example: man rsync even if it has lots of smaller features like symbolic links. So, agreed with @concretevitamin here.

Sounds good to me. We can remove the `cancel` part then.